2 failing tests to demonstrate issue #345 ExtendedMarkersView .. Limit

eobrienPilz opened the pull request:
2 Failing tests to demonstrate the problem described in issue #345 : https://github.com/eclipse-platform/eclipse.platform.ui/issues/345

jukzi commented:
Please rebase and add the added testclass to All...Tests, so that the CI executed the tests and fails

eobrienPilz committed:
2 failing tests to demonstrate issue #345 ExtendedMarkersView .. Limit #346

Squashed commit of the following:

commit 4dd1092b29dc33f1615fa78959535a523d24f0af
Author: Enda O Brien <E.OBrien@pilz.ie>
Date:   Fri Sep 20 17:52:36 2024 +0100

    2 failing tests to demonstrate issue #345 ExtendedMarkersView .. Limit #346

    Squashed commit of the following:

    commit 03b85b8753361c7678753a1972360351249524f6
    Merge: 693a6f14de c40b7a177e
    Author: eobrienPilz <58333578+eobrienPilz@users.noreply.github.com>
    Date:   Mon Mar 25 12:31:05 2024 +0000

        Merge branch 'master' into 345_ViewerFilterWithLimit

    commit 693a6f14de1c59da072a4fd515b98996535a3d9a
    Author: Enda O Brien <E.OBrien@pilz.ie>
    Date:   Tue Sep 20 13:01:09 2022 +0100

        2 failing tests to demonstrate issue #345 ExtendedMarkersView .. Limit
